regression/cbmc/byte_update13/test.desc

```diff
@@ -0,0 +1,8 @@
+CORE
```

Comment:
Tests: can we test behaviour in the other endianness somehow?

Reply:
Setting the `--arch` flags appropriately should do it.

Reply:
I don't think endianness actually matters in this test?